packages/aws-rfdk/lib/lambdas/nodejs/configure-spot-event-plugin/conversion.ts

```diff
@@ -27,7 +25,9 @@ export function convertSpotFleetRequestConfiguration(spotFleetRequestConfigs: Sp
     const convertedSpotFleetRequestProps: SpotFleetRequestProps = {
       AllocationStrategy: validateString(sfrConfigs.AllocationStrategy, `${group_name}.AllocationStrategy`),
       IamFleetRole: validateString(sfrConfigs.IamFleetRole, `${group_name}.IamFleetRole`),
-      LaunchSpecifications: convertLaunchSpecifications(sfrConfigs.LaunchSpecifications, `${group_name}.LaunchSpecifications`),
+      // Empty array needed for compatibility with SEP since it expects an array for the LaunchSpecifications property
+      LaunchSpecifications: [],
+      LaunchTemplateConfigs: sfrConfigs.LaunchTemplateConfigs ? validateLaunchTemplateConfigs(sfrConfigs.LaunchTemplateConfigs, `${group_name}.LaunchTemplateConfigs`) : undefined,
```

Comment:
Is it a valid use case if LaunchTemplateConfigs is empty?

Reply:
Good call out. We need either `LaunchSpecifications` or `LaunchTemplateConfigs` to be specified for the `request-spot-fleet` API. This is being enforced in our `SpotEventPluginFleet` construct, since it generates these configs based on the instance types and subnets provided, which we ensure we have at least one of each over [here](https://github.com/aws/aws-rfdk/blob/94988f2d1337bb7507ef9ea58695a27f72ab14f1/packages/aws-rfdk/lib/deadline/lib/spot-event-plugin-fleet.ts#L526-L537). So, right now users won't run into an issue where they have an empty configs list here. Since this file is focused on converting and not so much on validation, I'm thinking that we can just leave this as is. How does that sound?

Reply:
Sounds good to me